ttf2ugui.c

```diff
@@ -156,9 +156,9 @@ static void dumpFont(const UG_FONT * font, const char* fontFile, float fontSize,
 
 
   fprintf(out, "// For copyright, see original font file.\n");
-  fprintf(out, "\n#include \"ugui.h\"\n\n");
+  fprintf(out, "\n#include \"ui/ugui/ugui.h\"\n\n");
 
-  fprintf(out, "static __UG_FONT_DATA unsigned char fontBits_%s[%d][%d] = {\n", fontName, font->end_char - font->start_char + 1, bytesPerChar);
+  fprintf(out, "static UG_FONT_DATA unsigned char fontBits_%s[%d][%d] = {\n", fontName, font->end_char - font->start_char + 1, bytesPerChar);
```

Comment:
Why? Commit message would have been helpful here too.

Reply:
I guess we changed `__UG_FONT_DATA` in our repository to `UG_FONT_DATA` because GCC reserves all names that start with two underscores.

> [...] reserved names include all external identifiers (global functions and variables) that begin with an underscore (‘_’) and all identifiers regardless of use that begin with either **two underscores** or an underscore followed by a capital letter are reserved names.

https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html